src/main/kotlin/org/jetbrains/research/testspark/collectors/GeneratedTestsCollector.kt

```diff
@@ -0,0 +1,33 @@
+package org.jetbrains.research.testspark.collectors
+
+import com.intellij.internal.statistic.eventLog.events.EnumEventField
+import com.intellij.internal.statistic.eventLog.events.IntEventField
+import com.intellij.internal.statistic.service.fus.collectors.CounterUsagesCollector
+import org.jetbrains.research.testspark.collectors.data.Technique
+import org.jetbrains.research.testspark.data.CodeType
+
+/**
+ * This class represents a collector for tracking the event of generated tests.
```

Comment:
What exactly is tracked? The fact of test generation itself? Then the comment may be amended to `"... of test generation"`.